content/guides/translation-connection/step3-how-to-build-your-connector.md

```diff
@@ -179,16 +179,31 @@ For this action, Julia has to:
 
 After this last action, the bulk action will run and Julia will be notified in the Akeneo PIM notification bar once it is successfully executed.
 
-With these new attributes, your connector is now ready to go! Let's see how it'll behave on the other side of the connector. 
+With these new attributes, your connector is now ready to go! Let's see how it'll behave on the other side of the connector.
 
 ### How it works with your connector?
 
+**Find products with the enabled "translation queued" status**
+
 At regular intervals, your connector will need to retrieve products where the `Translation queued` attribute has been set to enable (set to `true`).
 
 You can do this by using our PIM [API filtering system](https://api.akeneo.com/documentation/filter.html#filter-on-product-values) on product attribute value.
 
+**Tell Julia that her translation project is "in progress"**
+
 Then, your connector needs to set this attribute to `false` and change the `Translation status` attribute to `IN PROGRESS` to indicate Julia that the translation is being processed by one of your translators.
```

Comment:
Why do you need to set the attribute "translation queued" to false? It could be good to precise why? I guess you are doing that in case an updated translation was needed?

Reply:
Sure, "translation queued" is to set that this product needs to be translated for the connector. So after this connector has retrieved the product, it is necessary to change this state in the PIM to prevent it from being taken into consideration when the connector re-processes PIM products.

Reply:
I add a comment in my documentation for that.